python/pyarrow/tests/test_fs.py

```diff
@@ -699,10 +699,7 @@ def test_get_file_info_with_selector(fs, pathfn):
 
         infos = fs.get_file_info(selector)
         if fs.type_name == "py::fsspec+s3":
-            # s3fs only lists directories if they are not empty, but depending
-            # on the s3fs/fsspec version combo, it includes the base_dir
-            # (https://github.com/dask/s3fs/issues/393)
-            assert (len(infos) == 4) or (len(infos) == 5)
+            len(infos) == 4
```

Comment:
```suggestion
            # s3fs only lists directories if they are not empty
            len(infos) == 4
```

(that first part is still valid)

Reply:
😬 🤦‍♀️  Thanks!